pylint_django/checkers/django_installed.py

```diff
@@ -20,8 +20,11 @@ class DjangoInstalledChecker(BaseChecker):
     }
 
     @check_messages('django-not-available')
-    def close(self):
+    def open(self):
         try:
             __import__('django')
         except ImportError:
-            self.add_message('F%s01' % BASE_ID)
+            # mild hack: this error is added before any modules have been inspected
+            # so we need to set some kind of value to prevent the linter failing to it
+            self.linter.set_current_module('project global')
```

Comment:
Could be `pylint_django` but either way is fine. IMO `project global` is something that is more puzzling, since it doesn't exist, but also isn't a valid Python module name.

Reply:
Agreed, `pylint_django` does make more sense. I'll change this.